cli/exec/exec.go

```diff
@@ -185,7 +186,10 @@ func execWithAxis(ctx context.Context, c *cli.Command, file, repoPath string, ax
 	}
 
 	// lint the yaml file
-	err = linter.New(linter.WithTrusted(true)).Lint([]*linter.WorkflowConfig{{
+	err = linter.New(
+		linter.WithTrusted(true),
+		linter.WithTrustedClonePlugins(constant.TrustedClonePlugins),
```

Comment:
INFO: exec normally will just cloning as it execs locally ... but we specify a default to be got to go ...